WebDriverAgentTests/IntegrationTests/XCElementSnapshotHelperTests.m

```diff
@@ -64,4 +64,18 @@ - (void)testParentMatchingType
   XCTAssertEqual(windowSnapshot.elementType, XCUIElementTypeWindow);
 }
 
+- (void)testParentMatchingOneOfTypes
+{
```

Comment:
Please split those two cases and add one more test for request of non existing type.